examples/30_extended/flow_id_tutorial.py

```diff
@@ -16,7 +16,14 @@
 import openml
 
 
+############################################################################
 # Activating test server
+#
+# .. warning:: This example uploads data. For that reason, this example
+#   connects to the test server at test.openml.org. This prevents the main
+#   server from crowding with example datasets, tasks, runs, and so on. The
+#   use of this test server can affect behaviour and performance of the
+#   OpenML-Python API.
```

Comment:
Could you please define the classifier in a different cell than you use to change the server?